### PR TITLE
Items crud operations

### PR DESCRIPTION
**Pull request by nefter**
Hello @virgyjimenez , ready to take your kind review

### CODE REVIEW

**Comment by nefter** [comment]
> Hello @virgyjimenez , thanks for reviewing my code.

Hello @virgyjimenez , ready to take your kind review

File: app/views/items/_form.html.erb [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
   <div>
     <%= form.label :title, style: "display: block" %>
-    <%= form.select :collection_id, options_for_select(Collection.all.collect {|x| [x.name, x.id]}, params[:collection_id]), {:include_blank => 'No Collection'} %>
+    <%= form.select :collection_id, options_for_select(Collection.all.collect {|x| [x.name, x.id]}, params[:collection_id]) %>
```
Comment (virgyjimenez) [inline app/views/items/_form.html.erb:21]: try `Collection.pluck(:name, :id)`

File: app/models/user.rb [hunk 2]
```diff
@@ -2,17 +2,18 @@
...
-  end
+  has_one_attached :avatar
+  # has_one_attached :avatar do |attachable|
+  #   attachable.variant :thumb, resize_to_limit: [100, 100]
+  # end
 
   def username
     # email = john@example.com -> john
-    email.split("@")[0].capitalize
+    email.split("@")[0].capitalize[0..8]
```
[the remark this answers is not in the document]
Comment (nefter) [reply app/models/user.rb:17]: Thank you, Vir, I will do the adjustment to include a username

File: app/views/items/_form.html.erb (hunk 1 above)
Comment (nefter) [reply app/views/items/_form.html.erb:21]: :exploding_head: Thanks Vir!

**Pull request merged**